vulcanai2/models/basenetwork.py

```diff
@@ -492,12 +491,15 @@ def save_model(self, save_path=None):
         pickle.dump(self.state_dict, open(state_dict_file_path, "wb"), 2)  # TODO: pretty sure this isn't necessary
 
     # TODO: update the state dict and push to the appropriate device.
+    # TODO: caitrin save the optimizers state dict? even though this is included with our instance?
     # https://pytorch.org/tutorials/beginner/saving_loading_models.html#what-is-a-state-dict
+    # TODO: implement, add in classification and input layers?
     @classmethod
-    def load_ensemble(cls, load_path):
+    def load_model(cls, load_path, load_ensemble=True):
```

Comment:
perhaps `load_stacks` or `complete_model_stack` is a better name than `load_ensemble`? ensemble may be confused with Snapshot ensemble or other types of networks that are ensembles